docs/specs/restore.yml

```diff
@@ -274,3 +274,65 @@ cache:
 outputs:
   docs/a.json: |
     {"conceptual":"<p>233333</p>"}
+---
+# build content in dependency repository
+# content in dependency repository:
+#   1. by default will not be built
+#   2. will be built when:
+#      1. explicitly turn on the inScope flag
+#      2. specify the dependency name in `build scope`
+repos:
+  https://docs.com/crrcontent/a:
+    - files:
+        docfx.yml: |
+          files:
+            - "docs/**/*"
+            - "crrb/**/*"
+          dependencies:
+            crrb:
+              type: git
+              remoteUrl: https://docs.com/crrcontent/b
```

Comment:
`msftbot` merged this too soon, consider:

- url: https://docs.com/crrcontent/b
- buildFiles: true

Reply:
already did in another PR for URL